PayloadPkg/FirmwareUpdate/CmdFwUpdate.c

```diff
@@ -51,6 +52,9 @@ FwUpdateCmdHandler (
   if(AsciiStrnCmp(CmdBuf, FLASH_DESCRIPTOR_LOCK_STR, AsciiStrLen(FLASH_DESCRIPTOR_LOCK_STR)) == 0) {
       *CmdProcessed  =  CMDI_TYPE_SPI_DESCRIPTOR_LOCK;
       Status = SetFlashDescriptorLock (CmdBuf, BufLen);
+  } else if(AsciiStrnCmp(CmdBuf, ARB_SVN_COMMIT_STR, AsciiStrLen(ARB_SVN_COMMIT_STR)) == 0) {

```

Comment:
why not use AsciiStrCmp () ?

Reply:
Currently we recommend command format as " {ARBSVNCOMMIT}
We would also need to support " {ARBSVNCOMMIT param1 param2..}
In this case we need to check for keyword using AsciiStrnCmp() to find keyword is present or not in string of words.

Reply:
got it.